docker-compose.yml

```diff
@@ -2,7 +2,9 @@ version: '3'
 
 services:
   db:
-    image: postgres:11.5
+    image: postgres:12
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
Is there a reason you update to postgres 12?

Reply:
Yes, after Django 4.0 update the docker setup fails with error:
```
Traceback (most recent call last):
  File "/binder/./setup.py", line 12, in <module>
    setup(
  File "/usr/local/lib/python3.12/site-packages/setuptools/__init__.py", line 104, in setup
    return distutils.core.setup(**attrs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/setuptools/_distutils/core.py", line 184, in setup
    return run_commands(dist)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/setuptools/_distutils/core.py", line 200, in run_commands
    dist.run_commands()
  File "/usr/local/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
    self.run_command(cmd)
  File "/usr/local/lib/python3.12/site-packages/setuptools/dist.py", line 967, in run_command
    super().run_command(command)
  File "/usr/local/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python3.12/site-packages/setuptools/command/test.py", line 223, in run
    self.run_tests()
  File "/usr/local/lib/python3.12/site-packages/setuptools/command/test.py", line 226, in run_tests
    test = unittest.main(
           ^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/unittest/main.py", line 104, in __init__
    self.parseArgs(argv)
  File "/usr/local/lib/python3.12/unittest/main.py", line 153, in parseArgs
    self.createTests()
  File "/usr/local/lib/python3.12/unittest/main.py", line 164, in createTests
    self.test = self.testLoader.loadTestsFromNames(self.testNames,
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/unittest/loader.py", line 207, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/unittest/loader.py", line 137, in loadTestsFromName
    module = __import__(module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/binder/tests/__init__.py", line 126, in <module>
    executor = MigrationExecutor(connection)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/migrations/executor.py", line 18, in __init__
    self.loader = MigrationLoader(self.connection)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/migrations/loader.py", line 58, in __init__
    self.build_graph()
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/migrations/loader.py", line 235, in build_graph
    self.applied_migrations = recorder.applied_migrations()
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/migrations/recorder.py", line 81, in applied_migrations
    if self.has_table():
       ^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/migrations/recorder.py", line 57, in has_table
    with self.connection.cursor() as cursor:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/backends/base/base.py", line 330, in cursor
    return self._cursor()
           ^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/backends/base/base.py", line 306, in _cursor
    self.ensure_connection()
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/backends/base/base.py", line 289, in ensure_connection
    self.connect()
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/backends/base/base.py", line 272, in connect
    self.init_connection_state()
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/backends/postgresql/base.py", line 309, in init_connection_state
    super().init_connection_state()
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/backends/base/base.py", line 239, in init_connection_state
    self.check_database_version_supported()
  File "/binder/.eggs/Django-4.2.11-py3.12.egg/django/db/backends/base/base.py", line 214, in check_database_version_supported
    raise NotSupportedError(
django.db.utils.NotSupportedError: PostgreSQL 12 or later is required (found 11.5).
```

Reply:
Changes reverted as we now force django 3 for local dev environment. see #242